tools/merge_licenses.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -o xtrace
+set -o errexit
+set -o nounset
+set -o pipefail
+
+output_path=${BUILD_WORKING_DIRECTORY}/MERGED_LICENSES
+# Truncate file.
+echo >$output_path
+
+for path in $(find ${BUILD_WORKING_DIRECTORY}/vendor/ -name LICENSE); do
+    echo adding $path;
+    # Trim working directory prefix from output.
+    echo "------------------------------------------------------------" >>$output_path
```

Comment:
might be more readable to use a block quote for this:
```bash
  cat <<EOF >> "${output_path}"
---------------
${path#${BUILD_WORKING_DIRECTORY}}
...
EOF

Reply:
Done